HW01/Ivan_Papushoi/ivan_papushoi_hw_01_t_03.rb

```diff
@@ -0,0 +1,9 @@
+def puts_2009_from_11_twos
+  solution_string = '2222 - 222 + 22 / 2 - 2'
+  evaluate_expression = 2222 - 222 + 22 / 2 - 2
+  consit_twos = solution_string.count '2'
```

Comment:
Instead of using variables you can move all these expressions to the methods. This approach would be more clear  
```suggestion
  def consit_twos
     solution_string.count '2'
   end
```

Reply:
Done